Expired/Views/ExpiringSoonView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ExpiringSoonView.swift
+//  Expired
+//
+//  Created by Karl Michael Reyes on 2023-03-22.
+//
+
+import SwiftUI
+
+struct ExpiringSoonView: View {
+    //    @State private var days  = 2
+    @EnvironmentObject var productStore: ProductStore
+    @Environment(\.managedObjectContext) private var viewContext
+    @AppStorage("notifyExpirySoonDate") private var days = 2
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    @EnvironmentObject var notificationVM: NotificationViewModel
+    
+    var body: some View {
+        VStack {
+            Text("\(days)")
+                .font(.system(size:45, weight:.bold, design: .rounded))
```

Comment:
```suggestion
                .font(.system(size: 45, weight: .bold, design: .rounded))
```

Apply consistent formatting: add space between `size:45` and `weight:.bold`